### PR TITLE
Feat well known endpoints

### PR DESCRIPTION
**Pull request by aspeteRakete**

### CODE REVIEW

File: dto/config.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import "github.com/teamhanko/hanko/config"
+
+// PublicConfig is the part of the configuration that will be shared with the frontend
+type PublicConfig struct {
+	Password config.Password `json:"password"`
+}
+
+// FromConfig Returns a PublicConfig from the Application configuration
+func FromConfig(config2 config.Config) PublicConfig {
+	return PublicConfig{Password: config2.Password}
```
Comment (lfleischmann) [inline dto/config.go:12]: Why `config2`? 

```suggestion
func FromConfig(config config.Config) PublicConfig {
	return PublicConfig{Password: config.Password}
```

Comment (aspeteRakete) [reply dto/config.go:12]: IntelliJ I guess.... :-)

File: handler/well_known.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	}, nil
+}
+
+func (h *WellKnownHandler) GetPublicKeys(c echo.Context) error {
+	keys, err := h.jwkManager.GetPublicKeys()
+	if err != nil {
+		return c.JSON(http.StatusInternalServerError, dto.NewApiError(http.StatusInternalServerError))
+	}
+	return c.JSON(http.StatusOK, keys)
+}
```
Comment (FreddyDevelop) [inline handler/well_known.go:29]: The returned json has a wrong format. See https://datatracker.ietf.org/doc/html/rfc7517#section-5
Expected: 
```
{
    "keys": [ {"e": "...", "kid": ""} ]
}
```

Got:
```
[ 
    {"e": "...", "kid": "..."}
]
```

**Review by FreddyDevelop — CHANGES_REQUESTED** [review]

File: handler/well_known.go (hunk 2 above)
Comment (aspeteRakete) [reply handler/well_known.go:29]: 🦅

File: handler/well_known_test.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
+package handler
+
+import (
+	"errors"
+	"github.com/labstack/echo/v4"
+	jwk2 "github.com/lestrrat-go/jwx/v2/jwk"
```
[the remark this answers is not in the document]
Comment (aspeteRakete) [reply handler/well_known_test.go:6]: suggestions?

[the remark this answers is not in the document]
Comment (lfleischmann) [reply handler/well_known_test.go:6]: `jwk3`? ...

Maybe use the default name for this and use `hankoJwk` for `our` package?

File: crypto/jwt/jwt.go [hunk 4]
```diff
@@ -19,27 +19,34 @@ type Generator interface {
 type generator struct {
 	signatureKey     jwk.Key
 	verificationKeys []jwk.Key
+	verKeys jwk.Set
```
Comment (FreddyDevelop) [inline crypto/jwt/jwt.go:22]: I don't see that it is used anywhere, so please delete it when its not used.

**Review by FreddyDevelop — CHANGES_REQUESTED** [review]

**Review by FreddyDevelop — APPROVED** [review]

**Pull request merged**